src/main/scala/org/broadinstitute/dsde/firecloud/service/TrialService.scala

```diff
@@ -259,58 +259,78 @@ final class TrialService
+              s" and a project could not be claimed: ${ex.getMessage}")
+            throw new FireCloudException("We could not process your enrollment. Please contact support. (Error 56)", ex)
+        }
+
       }
     }
 
     val saToken: WithAccessToken = AccessToken(googleDAO.getTrialBillingManagerAccessToken)
 
-    // 1. Check that the assigned Billing Project exists and contains exactly one member, the SA we used to create it
-    rawlsDAO.getProjectMembers(projectId)(saToken) flatMap { members: Seq[RawlsBillingProjectMember] =>
-      if (members.map(_.email.value) != Seq(googleDAO.getTrialBillingManagerEmail)) {
-        // TODO: for resiliency, try running this operation again with a new project
-        logger.warn(s"Cannot add user ${userInfo.userEmail} to billing project $projectId because it already contains members [${members.map(_.email.value).mkString(", ")}]")
-        Future(RequestCompleteWithErrorReport(InternalServerError, "We could not process your enrollment. Please contact support. (Error 60)"))
-      } else {
-        // 2. Add the user as Owner to the assigned Billing Project
-        rawlsDAO.addUserToBillingProject(projectId, ProjectRoles.Owner, userInfo.userEmail)(userToken = saToken) flatMap { _ =>
-          logger.info(s"[trialaudit] added user ${userInfo.userEmail} (${userInfo.id}) to project $projectId")
-          // 3. Update the user's Thurloe profile to indicate Enrolled status
-          val updatedTrialStatus = enrolledStatusFromStatus(status)
-          thurloeDao.saveTrialStatus(userInfo.id, userInfo, updatedTrialStatus) flatMap {
-            case Success(_) =>
-              logger.info(s"[trialaudit] updated user ${userInfo.userEmail} (${userInfo.id}) to state ${updatedTrialStatus.state.getOrElse("")}")
-              Future(RequestComplete(NoContent)) // <- SUCCESS!
-            case Failure(profileUpdateFail) => {
-              // We couldn't update trial status, so clean up
-              rawlsDAO.removeUserFromBillingProject(projectId, ProjectRoles.Owner, userInfo.userEmail)(userToken = saToken) map { _ =>
-                logger.warn(s"Enrolling user ${userInfo.userEmail} failed at profile update: ${profileUpdateFail.getMessage}. User has been backed out of billing project $projectId.")
-                RequestCompleteWithErrorReport(InternalServerError, s"We could not process your enrollment. Please try again later. (Error 70)")
-              } recover {
-                case bpFail: Throwable => {
-                  logger.warn(s"Enrolling user ${userInfo.userEmail} failed at profile update: ${profileUpdateFail.getMessage}. User is still in billing project $projectId due to cleanup failure: ${bpFail.getMessage}.")
-                  RequestCompleteWithErrorReport(InternalServerError, s"We could not process your enrollment. Please contact support. (Error 80)")
+    statusFuture flatMap { status =>
```

Comment:
this looks like a big diff but is mostly whitespace change - I wrapped this block of code in a `statusFuture flatMap {...` and indented it. Try ?w=1 in the github url to ignore whitespace changes.

Reply:
in the spirit of keeping changes small, I'm choosing not to touch the unchanged code ... do you feel strongly about this?